pkg/processor/deployment/deployment.go

```diff
@@ -293,7 +308,7 @@ func processEnv(name string, appMeta helmify.AppMetadata, c corev1.Container, va
 			if err != nil {
 				return c, errors.Wrap(err, "unable to set deployment value field")
 			}
-			c.Env[i].Value = fmt.Sprintf(envValue, name, containerName, strcase.ToLowerCamel(strings.ToLower(c.Env[i].Name)))
+			c.Env[i].Value = fmt.Sprintf(envValue, name, containerName, "env", strcase.ToLowerCamel(strings.ToLower(c.Env[i].Name)))
```

Comment:
please create issue and a separate PR for a bugfix

Reply:
ok I will make a separate pr :)